<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1987</width>
    <height>1027</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MS Serif</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-image: url(src/Assets/Menu.png);
}</string>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>400</y>
     <width>485</width>
     <height>427</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Style for header area  ####################################################*/ 

QCalendarWidget QWidget {
	 alternate-background-color: #fff;
}

/* style for top navigation area ###############################################*/ 

#qt_calendar_navigationbar {
    background-color: #fff;
	border: 2px solid  #B8E2FF;
	border-bottom: 0px;
	border-top-left-radius: 5px;
	border-top-right-radius: 5px;
}

/* style for month change buttons ############################################ */

#qt_calendar_prevmonth, 
#qt_calendar_nextmonth {
	/* border delete */
    border: none;  
    /* delete default icons */
	qproperty-icon: none; 
	
    min-width: 32px;
    max-width: 32px;
    min-height: 32px;
    max-height: 32px;

    border-radius: 5px; 
	/* set background transparent */
    background-color: transparent; 
	padding: 5px;
}

/* style for prev next month button ########################################### */
#qt_calendar_nextmonth {
	margin-right:5px;
	image: url(src/Assets/arrowRight.svg);
    /* qproperty-text: &amp;quot;&amp;gt;&amp;quot;; */
}
#qt_calendar_prevmonth {
	margin-left:5px;
	image: url(src/Assets/arrowLeft.svg);
}
#qt_calendar_prevmonth:hover, 
#qt_calendar_nextmonth:hover {
    background-color: #aaffff;
}

#qt_calendar_prevmonth:pressed, 
#qt_calendar_nextmonth:pressed {
    background-color: #55aaff;
}


/* Style for month button #################################### */


 #qt_calendar_monthbutton {
	width: 175px;
    color: #000;
	font-size: 32px;
	font-family: Inter;
	margin:5px 0px;
    border-radius: 5px;
	padding:0px 2px;
	
}

#qt_calendar_monthbutton:hover {
    background-color: #aaffff;
}

#qt_calendar_monthbutton:pressed {
    background-color: #55aaff;
}

/* Style for year input lineEdit ######################################*/

#qt_calendar_yearedit {
    min-width: 53px;
    color: #000;
    background: transparent;
	font-size: 13px;
}


/* Style for month select menu ##################################### */

#calendarWidget QToolButton QMenu {
     background-color: white;

}
#calendarWidget QToolButton QMenu::item {
	/*padding: 10px;*/
}
 #calendarWidget QToolButton QMenu::item:selected:enabled {
    background-color: #55aaff;
}

#calendarWidget QToolButton::menu-indicator {
	/* Remove toolButton arrow */
	image: none;
}

/* Style for calendar table ########################################## */
#qt_calendar_calendarview {
	/* Remove the selected dashed box */
    outline: 0px;

	border: 2px solid  #B8E2FF;
	border-top: 0px;
	border-bottom-left-radius: 5px;
	border-bottom-right-radius: 5px;
}

#qt_calendar_calendarview::item:hover {
   border-radius:5px;
	background-color:#aaffff;
}

#qt_calendar_calendarview::item:selected {
    background-color: #55aaff;
	border-radius:5px;
}

QCalendarWidget#qt_calendar_calendarview::item:hover {
   border-radius: 5px;
   background-color:#aaffff;
}

QCalendarWidget#qt_calendar_calendarview::item:selected {
    background-color: #55aaff;
    border-radius: 5px;
}
</string>
   </property>
   <property name="selectedDate">
    <date>
     <year>2023</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
   <property name="minimumDate">
    <date>
     <year>2023</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
   <property name="maximumDate">
    <date>
     <year>2023</year>
     <month>12</month>
     <day>31</day>
    </date>
   </property>
   <property name="firstDayOfWeek">
    <enum>Qt::Sunday</enum>
   </property>
   <property name="gridVisible">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QCalendarWidget::SingleSelection</enum>
   </property>
   <property name="horizontalHeaderFormat">
    <enum>QCalendarWidget::SingleLetterDayNames</enum>
   </property>
   <property name="verticalHeaderFormat">
    <enum>QCalendarWidget::NoVerticalHeader</enum>
   </property>
   <property name="navigationBarVisible">
    <bool>true</bool>
   </property>
   <property name="dateEditEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="aktivitasLabel_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>390</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Prioritas Kegiatan&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>460</y>
     <width>681</width>
     <height>361</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QScrollArea {
background-color: #fff;
}</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="container">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>679</width>
      <height>359</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
background-color: #fff;
}</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>681</width>
       <height>361</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="cardLayout"/>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="addJadwalButton">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>870</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>113</red>
         <green>219</green>
         <blue>242</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>113</red>
         <green>219</green>
         <blue>242</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>113</red>
         <green>219</green>
         <blue>242</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>113</red>
         <green>219</green>
         <blue>242</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>113</red>
         <green>219</green>
         <blue>242</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>113</red>
         <green>219</green>
         <blue>242</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>113</red>
         <green>219</green>
         <blue>242</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>113</red>
         <green>219</green>
         <blue>242</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>113</red>
         <green>219</green>
         <blue>242</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: #71DBF2;
border-radius: 22;
}

QPushButton:hover {
background-color: #aaffff;
}</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backbtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
